File: Advanced/PageAndRDL/SvgImage/C#/TestViewer/ViewerForm.cs
using GrapeCity.ActiveReports;
using GrapeCity.ActiveReports.Document;
using System;
using System.IO;
using System.Windows.Forms;

namespace TestViewer
{
	public partial class ViewerForm : Form
	{
		public ViewerForm()
		{
			InitializeComponent();
		}

		protected override void OnLoad(EventArgs e)
		{
			base.OnLoad(e);

			var reportFile = new FileInfo(@"..\..\..\..\Report\Svg.rdlx");
			var report = new PageReport(reportFile);
			var document = new PageDocument(report);

			viewer1.LoadDocument(document);
		}
	}
}

File: Advanced/PageAndRDL/SvgImage/VB.NET/TestViewer/ViewerForm.vb
Imports System.IO
Imports GrapeCity.ActiveReports.Document
Imports GrapeCity.ActiveReports

Public Class ViewerForm
	Protected Overrides Sub OnLoad(e As EventArgs)
		MyBase.OnLoad(e)

		Dim reportFile As FileInfo = New FileInfo("..\..\..\..\Report\Svg.rdlx")
		Dim report As PageReport = New PageReport(reportFile)
		Dim document As PageDocument = New PageDocument(report)

		viewer1.LoadDocument(document)
	End Sub
End Class
